perfkitbenchmarker/aws/aws_virtual_machine.py

```diff
@@ -143,6 +145,9 @@ def __init__(self, vm_spec):
     self.region = self.zone[:-1]
     self.image = self.image or GetImage(self.machine_type, self.region)
     self.user_name = FLAGS.aws_user_name
+    if self.machine_type in NUM_LOCAL_VOLUMES:
```

Comment:
I intentionally left striping the drives for another PR - if you want them in this one, that won't be too hard. What kind of flag do you want for that? Should we introduce a new one like --stripe_local_disks=True, or reuse --scratch_disk_type=striped_local, or just assume that benchmarks always want all the local drives striped?